app/src/main/res/values/strings.xml

```diff
@@ -170,4 +170,7 @@
   <string name="your_languages">Selected languages:</string>
   <string name="other_languages">Other languages:</string>
   <string name="no_items_msg">No items available</string>
+  <string name="sharing_message">The app is a lightweight piece of software reading bigger files stored on your device or SD card: once it is installed, you can select which additional content you would like to download (Wikipedia, Wiktionary, TED talks, etc.) and be ready for when your internet connection is bad. Download Kiwix Now:https://play.google.com/store/apps/details?id=org.kiwix.kiwixmobile </string>
+  <string name="sharing_body">Download Kiwix , The whole of Wikipedia will be on your device!</string>
```

Comment:
There an uncessary space between Kiwix and the comma and the T in the shouldn't be capitalised after the comma. Here's how applying English grammar would format the sentence: `Download Kiwix, the whole of Wikipedia will be on your device!`. 

On a related topic, downloading Kiwix does not give the whole of Wikipedia at the time of downloading the app. It can enable users to download the whole of Wikipedia if they wish (and have enough free space on their device, etc).

I'd suggest having a shorter `sharing_message` and a longer `sharing_body` (I'm assuming the message is more like the headline and the body the main text that appears after the `message`).